akka-docs/rst/scala/dispatchers.rst

```diff
@@ -153,6 +153,10 @@ And then using it:
 
```

Comment:
At line 45, use this as the example instead of the previous, i.e.

```
another example that uses the "thread-pool-executor":
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#fixed-pool-size-dispatcher-config
```